vendor.json

```diff
@@ -0,0 +1,5 @@
+[ {"url": "http://logstash.objects.dreamhost.com/maxmind/GeoLiteCity-2013-01-18.dat.gz", 
+  "sha1": "15aab9a90ff90c4784b2c48331014d242b86bf82" },
```

Comment:
I will fix that, but this should be considered for now just a PoC as there are still missing issues with the Bundler implementation to get the vendored files.